src/options.cpp

```diff
@@ -1333,6 +1360,21 @@ void show_options(bool ingame)
     delwin(w_options_tooltip);
 }
 
+template <typename maptype>
+void tile_loading_report(maptype tiletypemap, std::string label, std::string prefix) {
```

Comment:
Thanks for the const advice. I'll probably also move the function, especially since Jenkins says tilecontext isn't global and refuses to compile this as-is.